experiment/simulation/index.html

```diff
@@ -1,13 +1,225 @@
-<!DOCTYPE html>
-<html>
-<head>
-	<!-- Add CSS at the head of HTML file -->
-    <link rel="stylesheet" href="./css/main.css">
-</head>
-<body>
-		<!-- Your code goes here-->
-
-	<!-- Add JS at the bottom of HTML file --> 
-        <script src="./js/main.js"></script>
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Representation of Integers and their Arithmetic</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+</head>
+<body>
+    <div class="container justify-content-center">
+        <div class="grid">
+            <div class="row mx-4 my-3 my-sm-3 d-flex flex-row row-10 m-auto px-4">
+                <h1 id = "main-heading">Integer Arithmetic</h1>
+            </div>
+            <div class="row">
+                <div class = "message" id = "message" value="message"></div>                
+            </div>
+        </div>
+
+        <div class= "row">
+            <div class = "col-md-5">
+                <div class="col">

```

Comment:
Why do we need a separate div here just to apply the class col? Can't is be done in the parent div?